### PR TITLE
Fixing a few bugs

### PR DESCRIPTION
**Pull request by Vic-ST**
Fixing a few bugs for this.

---
This change is [Reviewable](https://reviewable.io/reviews/stevencopeland/stores/64)

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -25,21 +25,21 @@ $(document).ready(function() {
...
-        $('.game3').fadeIn(1000);
+    $(".gameiii").click(function() {
+        $(".game").hide(1000);
+        $(".game3").fadeIn(1000);
     });
-    $('.gameiv').click(function() {
-        $('.game').hide(1000);
-        $('.game4').fadeIn(1000);
+    $(".gameiv").click(function() {
+        $('.game").hide(1000);
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Unclosed string.

File: script.js [hunk 2]
```diff
@@ -25,21 +25,21 @@ $(document).ready(function() {
...
+        $('.game4").fadeIn(1000);
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Unclosed string.

File: script.js [hunk 3]
```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
 //make players
 var player1 = new player("#player1");
 var player2 = new player("player2");
-var player3 = new player('player3');
-var player4 = new player('player4');
-var ai1 = new player('AI1');
-var ai2 = new player('AI2');
-var ai3 = new player('AI3');
+var player3 = new player("player3");
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js [hunk 4]
```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
...
+var player4 = new player("player4");
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js [hunk 5]
```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
...
+var ai1 = new player("AI1");
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js [hunk 6]
```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
...
+var ai2 = new player("AI2");
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js [hunk 7]
```diff
@@ -55,11 +55,11 @@ var rollDice = function() {
...
+var ai3 = new player("AI3");
```
Comment (StevenCopeland) [inline script.js]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Too many errors. (28% scanned).

**Comment by Vic-ST** [comment]
Great!

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/64#-:-KhTGZuyFa2Y-i6sP4_t:btojbfd)*

**Comment by Vic-ST** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 7 unresolved discussions, some commit checks broke.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stevencopeland/stores/64)*

**Review by Vic-ST — COMMENTED** [review]
Awesome pull request! Fixes a lot of bugs.

File: script.js (hunk 1 above)
Comment (Vic-ST) [inline script.js:41]: __[jshint]__
* Unclosed string.

File: script.js (hunk 2 above)
Comment (Vic-ST) [inline script.js:42]: __[jshint]__
* Unclosed string.

File: script.js (hunk 3 above)
Comment (Vic-ST) [inline script.js:58]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js (hunk 4 above)
Comment (Vic-ST) [inline script.js:59]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js (hunk 5 above)
Comment (Vic-ST) [inline script.js:60]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js (hunk 6 above)
Comment (Vic-ST) [inline script.js:61]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Unclosed string.

File: script.js (hunk 7 above)
Comment (Vic-ST) [inline script.js:62]: __[jshint]__
* Expected an assignment or function call and instead saw an expression.
* Missing semicolon.
* Too many errors. (28% scanned).

**Pull request merged**